### ISSUE TITLE
scheduled task "mod_zoom\task\get_meeting_recordings" fails with recordingtype "shared_screen_with_speaker_view'"

### ISSUE THREAD

**Issue by samwitzig** [jrchamp/moodle-mod_zoom#599]
We unfortunately have the scheduled task "mod_zoom\task\get_meeting_recordings" failing: If the recording type is "shared_screen_with_speaker_view", we receive the error "ERROR:  value too long for type character varying(30)".

In the database scheme (https://github.com/ncstate-delta/moodle-mod_zoom/blob/main/db/install.xml -> line 127, we see that field recordingtype has 30 characters, "shared_screen_with_speaker_view" however has 31 characters.

**Comment by jrchamp** [comment]
Thanks @samwitzig! It looks like when we modified the data being stored in the field (2 months ago), we forgot to check the field's maximum size (last modified 3 years ago).

For the implementing developer, please make sure to use [Moodle's built-in XMLDB Editor tool](https://moodledev.io/general/development/tools/xmldb). We already know about a 35 character recording type, so maybe 50 would be a more comfortable limit.

**Comment by acquaalta** [comment]
Hi @jrchamp,
Thanks for @samwitzig and you for explaining the nature of the bug. We also encountered it.

My follow-up question is: What if I already installed the "Zoom meeting" plugin which has this 30 characters limitation in the "recordingtype" field? The relevant table is already created in Moodle DB, with that limitation in the field.
 Is there a way to fix the issue locally before you'll release an updated plugin version with the fix? I assume, of course, that the fix would also help these already have a previously installed a version of the plugin that has this bug (like 5.2.2, the latest one).

Thanks
